Source/WebKit/NetworkProcess/soup/WebSocketTaskSoup.cpp

```diff
@@ -63,16 +63,15 @@ WebSocketTask::WebSocketTask(NetworkSocketChannel& channel, const WebCore::Resou
     , m_cancellable(adoptGRef(g_cancellable_new()))
     , m_delayFailTimer(RunLoop::mainSingleton(), "WebSocketTask::DelayFailTimer"_s, this, &WebSocketTask::delayFailTimerFired)
 {
-    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GTK/WPE port
     auto protocolList = protocol.split(',');
     GUniquePtr<char*> protocols;
```

Comment:
What does this whole function body do? `protocols` is allocated on the stack and goes away at the end of this constructor. It's not clear to me that this whole body does anything useful but maybe I'm missing something?

Reply:
Oh, I should have expanded the diff. I thought it was the end of the constructor. My bad.